src/hotspot/cpu/riscv/nativeInst_riscv.cpp

```diff
@@ -113,32 +119,26 @@ address NativeFarCall::reloc_destination(address orig_address) {
   return stub_addr;
 }
 
-void NativeFarCall::set_destination(address dest) {
-  address addr = instruction_address();
-  assert(NativeFarCall::is_at(addr), "unexpected code at call site");
-  Unimplemented();
-}
-
-void NativeFarCall::verify() {
-  assert(NativeFarCall::is_at(instruction_address()), "unexpected code at call site");
+void RelocCall::verify() {
+  assert(RelocCall::is_at(instruction_address()), "unexpected code at call site");
 }
 
-void NativeFarCall::print() {
-  assert(NativeFarCall::is_at(instruction_address()), "unexpected code at call site");
-  tty->print_cr(PTR_FORMAT ": auipc,ld,jalr x1, offset/reg, ", p2i(addr_at(0)));
+void RelocCall::print() {
+  assert(RelocCall::is_at(instruction_address()), "unexpected code at call site");
+  tty->print_cr(PTR_FORMAT ": auipc,ld,jalr x1, offset/reg, ", p2i(instruction_address()));
 }
 
-bool NativeFarCall::set_destination_mt_safe(address dest, bool assert_lock) {
-  assert(NativeFarCall::is_at(addr_at(0)), "unexpected code at call site");
+bool RelocCall::set_destination_mt_safe(address dest, bool assert_lock) {
```

Comment:
Seens you need to merge latest HEAD and rebase. The `assert_lock` param of `NativeFarCall::set_destination_mt_safe` has been removed recently.

Reply:
Thanks for reminding, it's merged.